internal/callbacker/send_manager.go

```diff
@@ -273,35 +266,17 @@ func (m *sendManager) sendBatch(batch []*CallbackEntry) {
 		return
 	}
 
-	m.putInQuarantine()
-	m.handleQuarantineBatch(batch)
-}
+	until := time.Now().Add(m.delayDuration)
+	if time.Since(batch[0].Data.Timestamp) > m.expiration {
+		until = time.Unix(1<<63-1, 0)
```

Comment:
What does `time.Unix(1<<63-1, 0)` mean?

Reply:
Max time? 🤔

Reply:
Let's reuse this from the previous PR.

`time.Date(2999, time.January, 1, 0, 0, 0, 0, time.UTC)`
It's easier to understand

https://github.com/bitcoin-sv/arc/blob/55bb6cfd7cb0c99b292a85b62805646feac41371/internal/callbacker/quarantine_policy.go#L12